### DISCUSSION TITLE
怎么修改量算组件下面的label文字

### DISCUSSION THREAD

**Discussion by 1510171848** [zouyaoji/vue-cesium#470] [Q&A] [answered]
![Snipaste_2023-11-14_16-25-05](https://github.com/zouyaoji/vue-cesium/assets/43635418/4bff98c0-8365-40e1-b3d4-a7f1de4afd35)

**Comment by zouyaoji** [comment, marked as answer]
（1）通过修改各action的tooltip。 比如
```
distanceActionOpts: {
  //... 其他属性
tooltip: {
// ... 其他属性
tip: 'distance'
}
```
完整的在这儿
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/defaultOpts.ts#L16
（2）自定义lang传进去。
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/locale/lang/zh-hans.ts
https://zouyaoji.top/vue-cesium/#/zh-CN/component/vc-config-provider
